Fix overengineering

ferferga opened the pull request:
The scripts section is completely overngineered for some reason, while doing a copy is faster and easier.
Also fix for for WSA impossible to update

Checklist

- [X] Have tested the modifications

Howard20181 commented on scripts/Install.ps1:
```diff
@@ -0,0 +1,115 @@
+    exit
+}
+
+If (((Test-Path -Path "Styles","vendor.img","WsaProxy","system.img","CustomInstall","AppxManifest.xml","lxutil.dll","Microsoft.Web.WebView2.Core.dll","wslcoredeps.dll","Licenses","WsaClient","WSACodecs.dll","classicAppInstaller_WSA.sccd","userdata.vhdx","system_ext.img","networking.json","networking_schema.json","Images","libEGL.dll","WsaSettings.winmd","appcompatdb_schema.json","product.img","Ws ...
```
We shouldn't write a dead file list, it would be troublesome if the updated version files change, and this doesn't take into account the arm64 version.

ferferga replied:
This is exactly the script we're writing now. I guess that could go in another PR.

Howard20181 replied:
> This is exactly the script we're writing now. 

This list of files is generated dynamically

ferferga replied:
Sounds good to you if we remove the check altogether?

Howard20181 replied:
No

Howard20181 commented:
The current version will automatically uninstall if it is not compatible, we don't need this change

ferferga commented:
@Howard20181 For me, every time I run this script I get the specified error in the commit. I installed WSA from this repo yesterday (was using previously one from when Actions were a thing) and it completely nuked my Start menu icons.

As far as I understand it, we're looking for the normal WSA installation, not the custom one we want to install.

Try the script yourself in your system and see how it always ends up uninstalling WSA and reinstalling preserving data.

Howard20181 commented:
> Try the script yourself in your system and see how it always ends up uninstalling WSA and reinstalling preserving data.

It doesn't behave like this in my case, it can be upgraded normally every time

ferferga commented:
@Howard20181 Can you help me debug why?

Howard20181 commented:
How did you upgrade?
If the WSA image file is different from the previous location it cannot be upgraded, you need to overwrite to the same path of the previous installation

ferferga commented:
@Howard20181 My steps are as follow:

* Open WSA settings and deactivate. Close
* Go to Program Files/WSA (where I have this project's output) and clear the folder. Copy the newly built WSA version
* Run script using PowerShell as stated in the docs (some months ago, doing it in Windows Terminal would close it unexpectedly as it depends on vclibs)

Howard20181 commented:
> doing it in Windows Terminal would close it unexpectedly as it depends on vclibs

Are you using an older version of the build script?
The new version was installed a long time ago when it was determined that vclibs needed to be installed, so it doesn't affect anything except the first install

It is recommended to move the directory out of Program Files, as there will be permission issues

ferferga commented:
@Howard20181 I built exactly from this repo as of yesterday: https://github.com/EricFengFuuuu/MagiskOnWSA

Had some issues with Android 13, so I discovered this is the upgraded one.

Howard20181 commented:
That repository is so far behind us on commits that I don't think it's our problem

ferferga commented:
@Howard20181 I know, but I'm having that issue in this repository, not theirs :)

I'm moving the find output to an external file that could be read by the PowerShell script, so I think this PR is still relevant. I can remove the commit with the name fix and, if later on we find out that it's something that needs fixing, open a new PR for that (I included everything in one PR because I thought it would be easier to review for you, but it's true that their scope it's different).

Howard20181 commented:
> having that issue in this repository

move the directory out of Program Files

ferferga commented:
@Howard20181 Will try and report back (this time in a new issue)

I rebased my branch and now the file list is written to a ``filelist.txt`` file that gets read by the PowerShell script at runtime. Hopefully this time there are no blockers.

I also added a warning and the need of user intervention when a reinstall is going to be attempted, so at least the user has a chance of checking what's wrong before losing his start menu icons.

ferferga commented:
The changes won't appear in "File changes" tab until the PR is reopened, by the way

Howard20181 commented:
Can't reopen
![image](https://user-images.githubusercontent.com/40033067/211804214-b108f719-c4ca-407e-b883-d17176637a5f.png)

ferferga commented:
After a Windows reboot, upgrading worked fine repeteadly, even on Program Files, so there doesn't seem to be any permission issue in that folder.

To be sure, I moved it outside though.
Either way, problem solved! Thank you very much for your help!
